distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -533,231 +333,568 @@ public void testExportAfterDeletingAConfiguration() throws Exception {
      */
     @Test
     public void testWarningForAbsolutePathOutsideDdfHome() throws Exception {
-        FileUtils.deleteQuietly(getExportDirectory().toFile());
-        File systemProperties = new File(ddfHome + "/etc/system.properties");
-        File testFile = new File("../cat.txt");
+        resetInitialState();
+        
+        FileUtils.copyFile(SYSTEM_PROPERTIES.toFile(), new File(TEST_FILE));
+        System.setProperty(KEYSTORE_PROPERTY, ddfHome + File.separator + TEST_FILE);
+
+        String response = console.runCommand(EXPORT_COMMAND);
+        
+        assertThat(
+                String.format("Should not have been able to export to %s.", getExportDirectory()),
+                response, containsString(String.format("Failed to export all configurations to %s",
+                        getExportDirectory())));
+    }
+
+    /**
+     * Tests that absolute paths pointing inside ddfHome causes a warning
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testWarningForAbsolutePathInsideDdfHome() throws Exception {
```

Comment:
Now that we've combined all the test in the same file, we may want to add `Export` somewhere in the test method names related to the export command to make things more obvious.